isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/changelog.adoc

```diff
@@ -1,6 +1,12 @@
 [[changelog]]
 = Changelog
 
+*Änderungen IsyFact 3.1.0*
+
```

Comment:
Ich erhalte ganz oben immer noch den Hinweis mit ungültigen Zeichen im Branchnamen: 

The head ref may contain hidden characters: "feature/IFS-3821_Erg\u00E4nzung_Zweier_Issuer-URLs_in_Dokumentation_isy-security-(rel3.x)"